.github/workflows/plugins-ci.yml

```diff
@@ -0,0 +1,70 @@
+name: Plugin CI
+
+on:
+    workflow_call:
+
+jobs:
+    test:
+        runs-on: ${{ matrix.os }}
+
+        # output.COVERALLS is a variable to determine whether we should post to coveralls when all parallel test finish
+        outputs:
+            COVERALLS: ${{ steps.coveralls-trigger.outputs.COVERALLS_TRIGGER }}
+
+        strategy:
+            matrix:
+                node-version: [10, 12, 14, 16]
+                os: [macos-latest, ubuntu-latest, windows-latest]
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Use Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: ${{ matrix.node-version }}
+
+            - name: Install Dependencies
+              run: npm i --ignore-scripts
+
+            - name: Run Tests
+              run: npm test
+
+            - name: Coveralls Parallel
+              id: coveralls-parallel
+              continue-on-error: true
+              uses: coverallsapp/github-action@master
```

Comment:
I would add an `if` here to skip this step by default. An input parameter can turn it on

Reply:
Good idea!

Reply:
Added in https://github.com/fastify/workflows/pull/1/commits/3da8cf8459fd5e7f5a114790847fc31117ac558b